php_memcached.c

```diff
@@ -194,7 +194,8 @@ static inline php_memc_object_t *php_memc_fetch_object(zend_object *obj) {
 		php_error_docref(NULL, E_WARNING, "Memcached constructor was not called");    \
 		return;                                                                       \
 	}                                                                                 \
-	memc_user_data = (php_memc_user_data_t *) memcached_get_user_data(intern->memc);
+	memc_user_data = (php_memc_user_data_t *) memcached_get_user_data(intern->memc);  \
+	(void)memc_user_data;
```

Comment:
Well, ok let's do this, maybe add a comment like `/* avoid unused variable warning */` in the macro above the void line?